### ISSUE TITLE
Plugin gives loading error on OmegaT 5.4.0

### ISSUE THREAD

**Issue by msoutopico** [briacp/plugin-omt-package#6]
### Preconditions

Machine: Windows 10 (not tested in other OS's).
Plugin file  `plugin-omt-package-1.6.3.jar`  is in the user's `plugins` folder.

### Steps to reproduce

1. Install OmegaT 5.4.0
2. Launch OmegaT 5.4.0

### Expected results

1. The options to unpack, pack and unpack & delete project appear under the Project menu. 
2. Opens > Preferences > Plugins shows the OMT plugin.

### Actual results

None of the expected results above. 
When being launched, OmegaT gives the following error: 

![image](https://user-images.githubusercontent.com/3659409/102019433-52c22580-3d52-11eb-824f-bbd2b58ad420.png)

**Comment by msoutopico** [comment]
OmegaT.log says:
```20434: Error: Error load plugin from class 'net.briac.omegat.plugin.omt.ManageOMTPackage': NoClassDefFoundError: groovyjarjarcommonscli/ParseException (PLUGIN_LOAD_ERROR)
20434: Error: java.lang.NoClassDefFoundError: groovyjarjarcommonscli/ParseException 
20434: Error: 	at java.lang.Class.getDeclaredMethods0(Native Method) 
20434: Error: 	at java.lang.Class.privateGetDeclaredMethods(Class.java:2701) 
20434: Error: 	at java.lang.Class.privateGetMethodRecursive(Class.java:3048) 
20434: Error: 	at java.lang.Class.getMethod0(Class.java:3018) 
20434: Error: 	at java.lang.Class.getMethod(Class.java:1784) 
20434: Error: 	at org.omegat.filters2.master.PluginUtils.loadClass(PluginUtils.java:308) 
20434: Error: 	at org.omegat.filters2.master.PluginUtils.loadFromManifest(PluginUtils.java:271) 
20434: Error: 	at org.omegat.filters2.master.PluginUtils.loadPlugins(PluginUtils.java:117) 
20434: Error: 	at org.omegat.Main.main(Main.java:174) 
20434: Error: Caused by: java.lang.ClassNotFoundException: groovyjarjarcommonscli.ParseException 
20434: Error: 	at java.net.URLClassLoader.findClass(URLClassLoader.java:382) 
20434: Error: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:418) 
20434: Error: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) 
20434: Error: 	... 9 more 
```

**Comment by briacp** [comment]
Thanks Manuel,

This is fixed in [release 1.6.4](https://github.com/briacp/plugin-omt-package/releases/tag/v1.6.4). I actually fixed this issue a few months ago, but forgot to make a release with this fix, sorry.

**Comment by msoutopico** [comment]
I can confirm the error does not occur anymore. Thank you, Briac!
